cmd/testdata/tlsprofile_invalid.yaml

```diff
@@ -0,0 +1,2 @@
+featureGates:
```

Comment:
Instead of creating files for this, how about doing something like `internal/manifests/config/build_test.go`? We can have the content of the file there, write it before the test to a `tmp` folder, and read the content from there.